phpseclib/Crypt/Hash.php

```diff
@@ -69,6 +69,11 @@ class Hash
      */
     const PADDING_SHAKE = 3;
 
+    /** @var int|false */
+    public $etm;
+    /** @var string|false */
+    public $name;
```

Comment:
This should be in SSH2.php, as well, as they don't mean anything outside of SSH2.php.